conda-recipes/fairseq-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version="0.8.0" %}
+
+package:
+  name: fairseq
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/pytorch/fairseq 
+  git_rev: v{{ version }} 
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
If it's just a python package, do we need cxx compiler here?

Reply:
cxx compiler is needed to build the libbleu extension - https://github.com/pytorch/fairseq/tree/master/fairseq/clib/libbleu